packages/engine/Source/Shaders/EllipsoidVS.glsl

```diff
@@ -13,8 +13,8 @@ void main()
     vec4 p = vec4(u_radii * position, 1.0);
 
     v_positionEC = (czm_modelView * p).xyz;     // position in eye coordinates
-    gl_Position = czm_modelViewProjection * p;  // position in clip coordinates
-
+    vec4 positionEC = czm_modelView * position;
```

Comment:
This change is now multiplying by `czm_modelView` twice, which will affect performance. We could do the multiplication once, then re-use the result for the `vec3` variable.
```glsl
vec4 pEC = czm_modelView * p;
v_positionEC = pEC.xyz;
```